server/conf/env-var-docs.json

```diff
@@ -717,6 +717,11 @@
         "mode": "HIDDEN",
         "description": "Enables logic to populate more fields in OIDC logout requests.",
         "type": "bool"
+      },
+      "PROGRAM_CARD_IMAGES": {
+        "mode": "HIDDEN",
```

Comment:
Quick question on ADMIN_WRITEABLE: Is that intended so that some features are enabled for some deployments but not others? Or is it intended so that some deployments can enable the feature before we launch it for everyone?

I was thinking that program images are enabled for all deployments (if a deployment doesn't want to use them, that's fine -- we can still have it enabled), so I'm wondering what advantage ADMIN_WRITEABLE gives us.